src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -38,6 +40,17 @@ doServerInfo(RPC::JsonContext& context)
         context.params.isMember(jss::counters) &&
             context.params[jss::counters].asBool());
 
+    if (context.app.config().reporting())
+    {
+        Json::Value proxied = forwardToP2p(context);
+        if (proxied.isMember(jss::result) &&
+            proxied[jss::result].isMember(jss::info) &&
+            proxied[jss::result][jss::info].isMember(jss::load_factor))
+            ret[jss::info][jss::load_factor] =
+                proxied[jss::result][jss::info][jss::load_factor];
+        else
+            ret[jss::info][jss::load_factor] = 1;
```

Comment:
Is this testable? Can you add or update a unit test for it?

Reply:
No this is not until testable. Proxying requires running two servers and should be tested in integration/QA.

Reply:
Not really testable without adding some type of virtual proxy. This functionality (reporting) will be split into its own code base, so I think this stuff will be removed from rippled itself in that project.